.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - install_linter
       - run_tests
       # Change this to run_e2e_tests once we have stable algod binaries containing delta APIs
-      - run_e2e_tests_nightly
+      # - run_e2e_tests_nightly
```

Comment:
Chicken and egg problem here. I can't re-route the slack messaging until I fix the test. But the test is broken and getting slacked to the wrong channel. I've added a TODO to #1521 : 

<img width="425" alt="image" src="https://github.com/algorand/indexer/assets/291133/5f9ccb1f-6717-400c-94cf-e715fce4bebb">